### DISCUSSION TITLE
Add relatedIdentifierType "PMCID"

### DISCUSSION THREAD

**Discussion by KellyStathis** [datacite/datacite-suggestions#58] [Metadata Schema Suggestions]
### What is the problem that your suggestion solves?

PMCID is a critical identifier for US institutions who need manage NIH compliance. When linking to an article that is available open access in PubMed Central, it's better to use the PMCID because that will directly lead to the full text. Linking via the PMID requires an extra step to get to the actual article content.

### What solution might meet your needs?

I'd like PMCID to be added as a related identifier type. PMID is already listed as a type, but  PMID and PMCID are different identifiers for the metadata and article content respectively (https://nexus.od.nih.gov/all/2015/08/31/pmid-vs-pmcid-whats-the-difference/). Most articles will have both, and they will be different (see https://pubmed.ncbi.nlm.nih.gov/36092012/).

### What alternatives have you tried or considered?

We'll currently submit this to DataCite as PMID, even though that's not really correct.

### Is there anything else you would like to share?

*Suggestion was previously submitted by a user. For DataCite staff reference: https://docs.google.com/spreadsheets/d/1ItM7bYocla83TCYmHI9oZ9Kgt7X0Lznfv6Zls7Gmi88/edit?resourcekey=&gid=1473745305#gid=1473745305*

### What group(s) would benefit from your suggestion?

- [ ] DataCite members registering DOIs and metadata and administering accounts
- [ ] Developers building integrations with DataCite services
- [ ] Harvesters who index DataCite DOIs and metadata
- [ ] Individuals or institutions analyzing DataCite DOIs and metadata
- [ ] DataCite staff
- [ ] Other

**Comment by tedhabermann** [comment]
DataCite is about connections, so adding identifier types seems like it should be easy and essentially automatic... A link to [this page](https://nexus.od.nih.gov/all/2015/08/31/pmid-vs-pmcid-whats-the-difference/)  should be in the documentation.
